.gitignore

```diff
@@ -0,0 +1,2 @@
+
+scripts/__pycache__/
```

Comment:
Also add `*.pyc` files to catch any cached files that don't make it into the _pycache_ directory.